test/test_distributions.py

```diff
@@ -35,6 +35,13 @@ def __new__(cls, jax_dist, *params):
         return super(cls, T).__new__(cls, jax_dist, sp_dist, params)
 
 
+def _mvn_to_scipy(loc, cov, prec, tril):
+    jax_dist = dist.MultivariateNormal(loc, cov, prec, tril)
+    mean = device_get(jax_dist.mean)
+    cov = device_get(jax_dist.covariance_matrix)
```

Comment:
Why do we need `device_get` before passing to `osp.multivariate_normal`. Would a `DeviceArray` not work here?

Reply:
Oppss, it is legacy for debugging. It seems not necessary now.